torchgeo/datasets/cdl.py

```diff
@@ -58,6 +61,265 @@ class CDL(RasterDataset):
         (2008, "0610f2f17ab60a9fbb3baeb7543993a4"),
     ]
 
+    cmap = {
```

Comment:
Do we need to hardcode this in the dataset? It should be pulled automatically from the file itself right?

Reply:
I prefer to be explicit in case someone wants to copy paste the cmap or some version of the CDL tiffs don't include the cmap

Reply:
Hmm, I guess I'm on the fence. Explicit is always better than implicit, but this is a lot of lines of code just to be explicit. The attribute gets populated automatically as long as at least one year has this cmap (they all do so far as far as I know), and users can always access `dataset.cmap` if they want to copy or edit it.